ds-css/.pa11yci

```diff
@@ -1,9 +1,14 @@
 {
```

Comment:
Let's table the pa11y changes for another PR, it is out of the scope for adding build targets for the ds style guide